src/components/view/ListView.vue

```diff
@@ -61,18 +61,27 @@
     -->
 
     <span slot="name" slot-scope="text, record">
-      <div style="min-width: 120px">
-        <span v-if="$route.path.startsWith('/project')" style="margin-right: 5px">
-          <a-button type="dashed" size="small" shape="circle" icon="login" @click="changeProject(record)" />
-        </span>
-        <os-logo v-if="record.ostypename" :osName="record.ostypename" size="1x" style="margin-right: 5px" />
-        <console :resource="record" size="small" style="margin-right: 5px" />
+      <div style="min-width: 120px" >
+        <a-popover v-if="actions.length > 0" triggers="hover" placement="right">
```

Comment:
@davidjumani for read-only admin will this hide the name if there is nothing to be shown? the hover part LGTM but it's not clear perhaps add an explicit icon; what about components which don't have  name column as the first column?